companies/services-totalenergies.json

```
@@ -12,7 +12,8 @@
         "ELAN",
         "MINOL",
         "ACCESS",
-        "Total Access"
+        "Total Access",
+        "Heizöl"
```

That's a bit too generic in german, lets go for this instead: 
```suggestion
        "heizoel.totalenergies.de"
```